samples/interactive-tutorials/src/main/java/product/ImportProductsGcs.java

```diff
@@ -48,7 +48,7 @@ public static void main(String[] args) throws IOException, InterruptedException
     String gcsErrorBucket = String.format("%s/errors", gcsBucket);
     String gscProductsObject = "products.json";
     // TO CHECK ERROR HANDLING USE THE JSON WITH INVALID PRODUCT
-    // GCS_PRODUCTS_OBJECT = "products_some_invalid.json"
+    // gscProductsObject = "products_some_invalid.json";
```

Comment:
should this be `gcsProductsObject`?

Reply:
and if so, then I think the below method needs to get an updated argument as well.

Reply:
Refactored accordingly.